rtdb-limit-child-nodes/extension.yaml

```diff
@@ -120,6 +120,18 @@ params:
     validationErrorMessage: Path cannot have spaces.
     required: true
 
+  - param: SELECTED_DATABASE_INSTANCE
+    label: Selected Realtime Database instance
+    description: >
+      What is the selected Realtime Database used for this extension?
```

Comment:
```suggestion
    label: Realtime Database instance
    description: >
      Which Realtime Database instance do you want to limit?
```

Reply:
Thanks @kevinthecheung. I have updated the label and description. This matches updated added for https://github.com/firebase/extensions/pull/505